src/locale.js

```diff
@@ -51,7 +51,10 @@ export let date = {
   max: '${path} field must be at earlier than ${max}',
 };
 
-export let boolean = {};
+export let boolean = {
+  isTrue: '${path} field must be true',
+  isFalse: '${path} field must be false',
```

Comment:
you can get away with one message here: `isValue: '${path} field must be ${expected}', by adding the expected value to the test `params`

Reply:
Okay, done : )